pulseaudio-control.bash

```diff
@@ -302,22 +303,23 @@ function output() {
     if [ "$iconsLen" -ne 0 ]; then
         local volSplit=$((VOLUME_MAX / iconsLen))
         for i in $(seq 1 "$iconsLen"); do
-            if [ $((i * volSplit)) -ge "$curVol" ]; then
-                volIcon="${ICONS_VOLUME[$((i-1))]}"
+            if [ $((i * volSplit)) -ge "$VOL_LEVEL" ]; then
+                VOL_ICON="${ICONS_VOLUME[$((i-1))]}"
                 break
             fi
         done
     else
-        volIcon=""
+        VOL_ICON=""
     fi
 
     getNickname "$curSink"
 
     # Showing the formatted message
     if [ "$isMuted" = "yes" ]; then
-        echo "${COLOR_MUTED}${ICON_MUTED}${curVol}%   ${ICON_SINK}${nickname}${END_COLOR}"
+    	VOL_ICON=$ICON_MUTED	    
+	echo "${COLOR_MUTED}$(eval echo "$FORMAT")${END_COLOR}"
     else
-        echo "${volIcon}${curVol}%   ${ICON_SINK}${nickname}"
+	    echo "$(eval echo "$FORMAT")"
```

Comment:
please make sure you use 4 spaces for indentation (not only here)

Reply:
I didn't run a formatter on the whole file since it was messing up a lot. I just changed the indendation on the lines I added. Lmk if that was fine.